WebRoot/messages/ZaMsg_en_AU.properties

```diff
@@ -2792,3 +2792,5 @@ ERROR_InvalidRPLifetime = The range must be a positive integer.
 ERROR_RPExists = Policy "{0}" already exists.
 multiple_servers = multiple servers
 ERROR_INVALID_FILE_NAME = Invalid file name. Use browse button to select a valid file.
+ERROR_Feature_Not_Licensed = You are not currently not licensed for this feature.
```

Comment:
is it possible to merge both strings into a single property, so we can give translators flexibility around jumbling of words and sentences

Reply:
it's ok if we can't do this, it's a rare case scenario

Reply:
done